tools/nbs_tools/decode_image/__init__.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+from .decode_bayer import decode_bayer
+from .decode_jpeg import decode_jpeg
+
+from .fourcc import fourcc
+
+
+def decode_image(data, format):
+
+    if format in [fourcc("JPBG"), fourcc("JPRG"), fourcc("JPGR"), fourcc("JPGB")]:
+        return decode_bayer(data, format)
```

Comment:
This should decode the jpeg before calling `decode_bayer`. This function should also handle raw bayer formats

Reply:
It actually just permutes bayer, so in theory, you don't even have to decode a raw bayer using this